hack/pre-release.sh

```
@@ -424,14 +438,15 @@ if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
-while getopts "bcdg" opt; do
-  case "$opt" in 
+# This sets the CVP, baseimage, and other flags
+while getopts "bcdgk" opt; do
+  case "$opt" in
     c) CVP=true;;
     b) BASEIMAGE=true;;
     d) DISTGIT=true;;
     g) GITLAB=true;;
-  esac 
+    k) KEEP_BUNDLE_CHANGES=true;;
+  esac
```

Shellcheck (0.11.0)

[warning] Invalid flags are not handled. Add a *) case.

(SC2220)